thoth/storages/graph/postgres.py

```diff
@@ -3475,12 +3475,13 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
-    def get_kebechet_github_installations_active(self) -> List[str]:
-        """Return all of the repositories wiht the active Kebechet installation."""
+    def get_kebechet_github_installations_per_is_active(self) -> List[str]:
```

Comment:
also @pacospace, what should be the return type? According to documentation it is `Dict[str, List[Any]]`,
should it be dict of repo_names with enabled managers then?

Reply:
your filter in this case, is `is_active` therefore you should have something like 

{'active': ['repo/1', 'repo/2'], inactive: [] }

but that is not what you want I think:

You want to have all repos that have active Kebechet, therefore I see two options 

1. => `get_kebechet_github_installations_all()` (Type one naming)


```
def get_kebechet_github_installations_per_is_active(self, is_active: Optional[bool] = None) -> List[str]:
"""Get all repositories names with Kebechet installation."""
        with self._session_scope() as session:
            query = session.query(KebechetGithubAppInstallations)

              if is_active:
                     query = query.filter(KebechetGithubAppInstallations.is_active.is_(is_active))

             result = query.all()

         return [m.repo_name for m in result]
```

2. => `get_active_kebechet_github_installations_repos()` (Type seven naming)

```
def get_active_kebechet_github_installations_repos(self) -> List[str]:
"""Get all active repositories names with active Kebechet installation."""
        with self._session_scope() as session:
            active_managers = (
                session.query(KebechetGithubAppInstallations)
                .filter(KebechetGithubAppInstallations.is_active.is_(is_active))
                .all()
            )
         return [m.repo_name for m in active_managers]
```


naming convention for `get_kebechet_github_installations_count_per_is_active` is not respected actually. Whatever is returning type number should be should have _count at the end => `get_active_kebechet_github_installations_repos_count_all` @saisankargochhayat (Type six naming)

in that case we can also generalize it actually:

```
 def get_kebechet_github_installations_repos_count_all(self,  is_active: Optional[bool] = None) -> int:
"""Get the number of repos with Kebechet installation."""
```

Reply:
Okay, changed accordingly to the type 7. Also I've changed the return value to be of { <repo>: [<managers>] }
plz have a l00k